package.json

```diff
@@ -62,13 +62,13 @@
   "scripts": {
     "build": "bash ./bin/xpi.sh",
     "eslint": "eslint addon test --ext jsm --ext js --ext json",
+    "firefox": "export XPI=dist/linked-addon.xpi && npm run build && node run-firefox.js",
+    "harness_test": "export XPI=dist/linked-addon.xpi && mocha test/functional_tests.js --retry 2 --reporter json",
     "lint": "npm-run-all lint:*",
     "lint:addons-linter": "# actually a post build test:  bin/addonLintTest ' + require('./package.json').name",
-    "lint:eslint": "npm eslint",
+    "lint:eslint": "npm run eslint",
```

Comment:
This was the only changed line. I think the other 2 lines were just some auto A-z sorting by npm.

Reply:
Thanks!  Maybe look over the main eslintrc, and see if you think that's a good set of stuff, or if there is more to add?

Reply:
Done. Beefier ESLint proposal submitted in #37 PR.